scrypt_test.go

```diff
@@ -140,7 +143,7 @@ func TestCalibrate(t *testing.T) {
 		}
 		if dur < timeout/2 {
 			t.Errorf("%d. GenerateFromPassword was too fast (wanted around %s, got %s) with %#v.", testNum, timeout, dur, p)
-		} else if timeout*2 < dur {
+		} else if timeout+timeout/2 < dur {
 			t.Errorf("%d. GenerateFromPassword took too long (wanted around %s, got %s) with %#v.", testNum, timeout, dur, p)
```

Comment:
Looks good, but one last ask: can you change the error message to detail what "around" means - e.g. "expected between {lower bound} and {upper bound}".

Reply:
Yes of course